resources/safe-labels-pods.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: policies.kubewarden.io/v1
```

Comment:
quibble: Would prefer for this file to be named `policy-safe-labels-pods.yaml`, or `safe-labels-pods-policy.yaml`.